deploy/devel/deployment.patch.yaml

```diff
@@ -6,7 +6,7 @@
 - op: replace
   path: /spec/template/spec/containers/0/env/1/value
   # this resolves to the ip of the host running minikube
```

Comment:
can we update the comment here?

Reply:
Sure, what do you want mentioned there?

Reply:
Something like

```
This patches the ALLOWEDDOMAINS env variable to allow connections from all the hostnames and IPs currently used in dev environments.
TODO: make this more generic (switching to Helm might help)
```

Feel free to rewrite as it sounds best to you!